tests/cdu/generator/test_llm_command_generator.py

```diff
@@ -0,0 +1,42 @@
+import uuid
+from typing import Dict
+
+import pytest
+from _pytest.tmpdir import TempPathFactory
+
+from rasa.dialogue_understanding.generator.llm_command_generator import (
+    LLMCommandGenerator,
+)
+from rasa.engine.storage.local_model_storage import LocalModelStorage
+from rasa.engine.storage.resource import Resource
+from rasa.engine.storage.storage import ModelStorage
+
+
+@pytest.fixture(scope="session")
+def resource() -> Resource:
+    return Resource(uuid.uuid4().hex)
+
+
+@pytest.fixture(scope="session")
+def model_storage(tmp_path_factory: TempPathFactory) -> ModelStorage:
+    return LocalModelStorage(tmp_path_factory.mktemp(uuid.uuid4().hex))
+
+
+@pytest.mark.parametrize(
```

Comment:
STY: I'd split this up into separate tests. keeps tests simpler and easier to reason about